Lib/test/test_embed.py

```diff
@@ -70,15 +70,18 @@ def setUp(self):
             expecteddir = os.path.join(support.REPO_ROOT, builddir)
         else:
             exepath = os.path.join(builddir, 'Programs')
-            expecteddir = os.path.join(support.REPO_ROOT, 'Programs')
+            expecteddir = os.path.join(builddir, 'Programs')
```

Comment:
There's no need for expecteddir at all if the calculation is identical to exepath - it's only used for ensuring that these tests run in-tree because they need(ed) to.

Reply:
Sure, I'll remove it.